Fix build with qt5.3
Index: qtcurve/qt5/style/qtcurve_api.cpp
===================================================================
--- qtcurve.orig/qt5/style/qtcurve_api.cpp	2015-02-08 20:16:47.671846314 +0530
+++ qtcurve/qt5/style/qtcurve_api.cpp	2015-02-08 20:16:47.667846315 +0530
@@ -67,25 +67,6 @@
 
 #include <QDebug>
 
-extern QString (*qt_filedialog_existing_directory_hook)(
-    QWidget *parent, const QString &caption, const QString &dir,
-    QFileDialog::Options options);
-
-extern QString (*qt_filedialog_open_filename_hook)(
-    QWidget *parent, const QString &caption, const QString &dir,
-    const QString &filter, QString *selectedFilter,
-    QFileDialog::Options options);
-
-extern QStringList (*qt_filedialog_open_filenames_hook)(
-    QWidget * parent, const QString &caption, const QString &dir,
-    const QString &filter, QString *selectedFilter,
-    QFileDialog::Options options);
-
-extern QString (*qt_filedialog_save_filename_hook)(
-    QWidget *parent, const QString &caption, const QString &dir,
-    const QString &filter, QString *selectedFilter,
-    QFileDialog::Options options);
-
 namespace QtCurve {
 
 void
@@ -163,13 +144,6 @@
         opts.menuBgndAppearance = APPEARANCE_FLAT;
     }
 
-    if (opts.useQtFileDialogApps.contains(appName)) {
-        qt_filedialog_existing_directory_hook = 0L;
-        qt_filedialog_open_filename_hook = 0L;
-        qt_filedialog_open_filenames_hook = 0L;
-        qt_filedialog_save_filename_hook = 0L;
-    }
-
     QCommonStyle::polish(app);
     if (opts.hideShortcutUnderline) {
         app->installEventFilter(m_shortcutHandler);
